.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
 dist: trusty
```

Comment:
Lets move this to bionic, trusty is ancient. You also need to rebase. And when I did this in the fork I used 9.2.8.0 for jruby only, we probably want to do that.